src/BenchmarkDotNet/Code/CodeGenerator.cs

```diff
@@ -95,6 +97,24 @@ internal static string Generate(BuildPartition buildPartition)
             return benchmarkProgramContent;
         }
 
+        private static string GetCustomEnvInfo(DeclarationsProvider provider)
+        {
+            if (provider.CustomEnvInfoMethodName is null)
+            {
+                return "Array.Empty<string>()";
```

Comment:
Could you please use the full name? 

```suggestion
                return "System.Array.Empty<string>()";
```

Reply:
OK, let's try this suggestons feature. :)